addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/handler/FreeboxHandler.java

```
@@ -175,15 +188,15 @@ private boolean authorize(boolean useHttps, String fqdn, String apiBaseUrl, Stri
             loginManager.setAppToken(configuration.appToken);
             loginManager.openSession();
             return true;
-        } catch (FreeboxException | InterruptedException e) {
-            logger.error("{}", e.getMessage());
+        } catch (Exception e) {
```

My intention was to catch all exceptions to display an appropriate log message.